pages/ActivityStreams/Overview.md

```diff
@@ -0,0 +1,55 @@
+---
+name: Overview
+route: /ActivityStreams/Overview
+menu: Activity Streams
+---
+
+# Activity Streams Overview
+
+Content shared via the DSNP will generally consist of URIs pointing to documents containing Activity Streams JSON objects.
+For the purposes of the DSNP, only a subset of the [Activity Streams 2.0](https://www.w3.org/TR/activitystreams-core/) specification must be supported by default.
+Implementers may choose to support more of the Activity Streams standards as they see fit, but as a warning, these standards may include ambiguities which could lead to different and incompatible implementations of certain features.
+Implementers who extend their support for Activity Streams objects beyond the minimal subset defined here do so at their own risk.
+
+Additionally, some aspects of the Activity Streams standard may be modified or extended in the context of the DSNP to support specific validations or features necessary in a distributed context.
+These deviations from the standard will be thoroughly documented here.
+
+## Specification Status
+
+| Version | Status |
+---------- | ---------
+| 0.1     | Tentative |
+
+## Purpose
+
+1. Identify what aspects of the existing Activity Streams specification is supported, encouraged or enforced in the DSNP.
+1. Specify any extensions or modifications of the Activity Streams specification that must be supported or required by the DSNP.
+
+## Linking Requirements
+
+* All broadcast messages link to activity stream JSON, not any other form of content.
+* All reply messages link to an activity stream JSON, not any other form of content.
+
+## Minimum Subset
+
+At minimum, all clients implementing the DSNP must support presentation for the following message types from the Activity Streams vocabulary: `"Link"`, `"Note"` and `"Person"`.
+It is also strongly encouraged to support `"Audio"`, `"Image"`, `"Mention"`, `"Profile"`, `"Tombstone"` and `"Video"` where ever possible.
+
+## Extensions
+
+In addition to the standards defined by the Activity Streams specifications, some extensions of Activity Streams are also required by the DSNP due to its decentralized context.
+
+### Required Keys and Values
+
+Certain suggested properties of objects in the Activity Streams standard are required for validation by the DSNP.
```

Comment:
It would be useful to link to where these required properties are specified in this spec.

Reply:
I like this idea. Having these be listed out and linked here. @rosalinekarr perhaps you can catch it Monday.

Reply:
Actually this could be covered by https://github.com/LibertyDSNP/spec/issues/58